### PR TITLE
Allow publishing on GitHub.com

### PR DESCRIPTION
**Pull request by fregante**
Like https://github.com/pixiebrix/eslint-config-pixiebrix/commit/0ccb477dde429c9aa72a387f99316e246535ba67

This package hasn't been published in almost a year, let's make it easier to do it

### CODE REVIEW

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      - run: npm ci || npm install
+      - uses: fregante/setup-git-user@v2
+      - name: Create version
+        # Get the generated version, this enables support for keywords: `npm version patch`
+        run: |
+          VERSION="$(npm version "${{ github.event.inputs.Version }}")"
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (fregante) [inline .github/workflows/npm-publish.yml:36]: @twschiller we need a NPM_TOKEN with write access to webext-messenger. Visit:

https://www.npmjs.com/settings/tschiller/tokens

Comment (twschiller) [reply .github/workflows/npm-publish.yml:36]: It looks like it was set but expired. It should be good to go now

@grahamlangford links to notes on how to create the token: https://www.notion.so/pixiebrix/Creating-Updating-NPM-Publishing-Tokens-b8f083afbf634ffc923337c8bb44091d?pvs=4

Comment (fregante) [reply .github/workflows/npm-publish.yml:36]: For him to be able to do the renewal, he also needs to be listed as a owner in the package, but I only see you on npm: https://www.npmjs.com/package/webext-messenger

**Comment by fregante** [comment]
The publishing worked, but the branch protections prevented this from working. We had talked about this for the before, I think, webext-messenger isn't a Pixiebrix product so it doesn't need this kind of protection. The extension itself already has the appropriate branch protections.

image

**Pull request merged**